src/styles/blocks/content.scss

```diff
@@ -0,0 +1,121 @@
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding-top: 50px;
+  gap: 48px;
```

Comment:
Flex gap is not supported by some browser versions.

You don't need to fix it, but be aware of it in the future work.